kedro/templates/project/{{ cookiecutter.repo_name }}/setup.cfg

```
@@ -1,3 +1,7 @@
+[flake8]
+max-line-length=88
+extend-ignore=E203
```

Maybe? As per what [Black suggests](https://black.readthedocs.io/en/stable/compatible_configs.html#flake8)
```suggestion
extend-ignore=E203,W503
```

---

I guess it's optional. Both `W503` and it's counterpart, `W504`, are ignored by default (https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-ignore) as of https://pycodestyle.pycqa.org/en/latest/developer.html#id9, hence the minimal-compatible config here: https://black.readthedocs.io/en/stable/the_black_code_style.html#line-length (which is what I've used)